acceptance/lib/helper.rb

```diff
@@ -259,7 +285,34 @@ def https_request(url, request_method, cert, key, body = nil)
     response = http.request(request)
   end
 
-
+  def hup_server(host = master, timeout = 30)
+    pid=on(host, 'cat /var/run/puppetlabs/puppetserver/puppetserver').stdout.chomp
```

Comment:
you could do `puppet master --configprint rundir` + `puppetserver/puppetserver` I think.